contracts/standard/arbitration/CentralizedCourt.sol

```diff
@@ -0,0 +1,231 @@
+ */
+
+contract CentralizedCourt is Arbitrator {
+
+    address public owner = msg.sender;
+    uint arbitrationPrice; // Not public because arbitrationCost already acts as an accessor.
+    uint constant NOT_PAYABLE_VALUE = (2**256-2)/2; // High value to be sure that the appeal is too expensive.
+
+    struct VoteCounter {
+        uint winningChoice; // The choice which currently has the highest amount of votes. Is 0 in case of a tie.
+        uint winningCount;  // The number of votes for winningChoice. Or for the choices which are tied.
+    }
+
+    struct Vote {
+        address account; // The juror who casted the vote.
+        uint ruling;     // The ruling which was given.
+    }
+
+    struct Dispute {
+        Arbitrable arbitrated;
+        uint choices;
+        uint fee;
+        DisputeStatus status;
+        mapping (address => bool) hasVoted;
+        mapping (address => bool) canCollectFee;
+        mapping (uint => uint) voteCount; // voteCount[choice] is the number of votes for choice.
+        VoteCounter voteCounter;
+        Vote[] votes;
+        uint deadline;
+    }
+
+    modifier onlyOwner {
+        require(msg.sender==owner);
+        _;
+    }
+
+    Dispute[] public disputes;
+    address[] public members;
+    mapping (address => bool) public isMember;
+
+    uint constant public MAX_MEMBER_COUNT = 50;
+
+    modifier onlyCourtMember() {
+        require(isMember[msg.sender]);
+        _;
+    }
+
+    modifier whoHasNotVoted(uint _disputeID) {
+        require(!disputes[_disputeID].hasVoted[msg.sender]);
+        _;
+    }
+
+    modifier beforeDeadline(uint _disputeID) {
+        require(now < disputes[_disputeID].deadline);
+        _;
+    }
+
+
+    /** @dev Constructor. Set the initial arbitration price.
+     *  @param _arbitrationPrice Amount to be paid for arbitration.
+     *  @param _members Array of addresses allowed to rule.
+     */
+    constructor (uint _arbitrationPrice, address[] _members) public {
+        require(_members.length <= MAX_MEMBER_COUNT);
+        for(uint8 i = 0; i < _members.length; i++){
+            require(!isMember[_members[i]] && _members[i] != address(0));
+            isMember[_members[i]] = true;
+        }
+        arbitrationPrice = _arbitrationPrice;
+        members = _members;
+    }
+
+
+    /** @dev Create a dispute. Must be called by the arbitrable contract.
+     *  Must be paid at least arbitrationCost().
+     *  @param _choices Amount of choices the arbitrator can make in this dispute. When ruling ruling<=choices.
+     *  @param _extraData Can be used to give additional info on the dispute to be created.
+     *  @return disputeID ID of the dispute created.
+     */
+    function createDispute(uint _choices, bytes _extraData) public payable returns(uint disputeID)  {
+        require(msg.value >= arbitrationPrice);
+
+        disputeID = disputes.length++;
+        Dispute storage dispute = disputes[disputeID];
+        dispute.deadline = now + 3 days;
```

Comment:
Avoid magic constant.
The time for deadline can be a parameter put in the constructor.